src/auth/auth.service.ts

```diff
@@ -133,6 +135,54 @@ export class AuthService {
         return tokens;
     }
 
+    async revokeRefreshToken(body?: RevokeRTDto) {
+        const { userId, email } = body;
+
+        if (userId && email)
+            throw new BadRequestException(
+                "Please provide either userId or email, not both",
+            );
+
+        if (userId) {
+            try {
+                await this.prisma.user.update({
+                    where: {
+                        id: userId,
+                    },
+                    data: {
+                        refreshToken: null,
+                    },
+                });
+            } catch (error) {
+                throw new NotFoundException({
+                    statusCode: 404,
+                    message: `User by "${userId}" cannot be found.`,
```

Comment:
```suggestion
                    message: `User by '${userId}' cannot be found.`,
```

just look better on swagger 
![image](https://github.com/chingu-x/chingu-dashboard-be/assets/6191116/3a60266b-22bc-4a70-ad0f-008247ab4813)
vs
![image](https://github.com/chingu-x/chingu-dashboard-be/assets/6191116/425741fb-235f-421d-9061-90bafffdd748)